help/en/html/hardware/pi/index.shtml

```diff
@@ -44,6 +44,10 @@
       "Pi2 model B (&copy; www.raspberrypi.org)"> <a name=
       "hardware" id="hardware"></a>
 
+      <p class="noted">The Raspberry Pi can also be used to run a full version of JMRI under the
```

Comment:
This statement is wrong.  To use the Raspberry Pi hardware support, you must first install JMRI.  This page is about configuration of that hardware support within JMRI, in other words JMRI installation is a pre-requisite for the hardware support discussed on this page.

This is no different than ANY other system we support.